crates/uv/tests/common/mod.rs

```diff
@@ -304,19 +304,19 @@ pub fn bootstrapped_pythons() -> Option<Vec<PathBuf>> {
         .parent()
         .unwrap()
         .to_path_buf();
-    let boostrap_dir = if let Some(boostrap_dir) = env::var_os("UV_BOOTSTRAP_DIR") {
```

Comment:
boostrap_dir, where you save all your 👻s

Reply:
lol I typo that one all the time